### PR TITLE
added Gatsby

### PR DESCRIPTION
**Pull request by caioreis123**
Please leave me a comment if changes are needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -320,6 +339,14 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: There is no formal signup required. Create five PRs on either [Adobe](https://github.com/adobe) or [Magento](https://github.com/magento) repositories during the month of October. Adobe team will file an issue in November in [this repo](https://github.com/adobe/squashtoberfest) listing out qualifiying contributors' handles in the file with further instructions to get the swags.
 - **Notes**: See more on [official blog](https://community.magento.com/t5/Magento-DevBlog/Squashtoberfest-2019/ba-p/143694) and their [Squashtoberfest repo](https://opensource.adobe.com/squashtoberfest/)
 
+#### **Gatsby** (Level 1 swag)
```
Comment (travi) [inline README.md]:
```suggestion
#### **Gatsby** (Level 2 swag)
```

File: README.md [hunk 2]
```diff
@@ -60,6 +60,17 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
 
+### G
+
+### **Gatsby**
+
+- **Swag**: T-shirt, Stickers, Socks, Hats, Pants
+- **Requirements**:
+  - 1 merged pull requests: Level 1 swag
+  - 5 merged pull requests: Level 2 swag
+- **How to sign up**: Just submit your PRs to [Gatsby] (https://github.com/gatsbyjs/store.gatsbyjs.org) and log into their [store](https://store.gatsbyjs.org/) with your GitHub account.
```
Comment (crweiner) [inline README.md]: Be sure to not have a space between the `[brackets](parenthesizes)`, or else the link is not formatted correctly when Jekyll builds the page.

```suggestion
- **How to sign up**: Just submit your PRs to [Gatsby](https://github.com/gatsbyjs/store.gatsbyjs.org) and log into their [store](https://store.gatsbyjs.org/) with your GitHub account.
```

Please check your contribution as this appears in multiple places. :+1:

File: README.md [hunk 3]
```diff
@@ -320,6 +339,14 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: There is no formal signup required. Create five PRs on either [Adobe](https://github.com/adobe) or [Magento](https://github.com/magento) repositories during the month of October. Adobe team will file an issue in November in [this repo](https://github.com/adobe/squashtoberfest) listing out qualifiying contributors' handles in the file with further instructions to get the swags.
 - **Notes**: See more on [official blog](https://community.magento.com/t5/Magento-DevBlog/Squashtoberfest-2019/ba-p/143694) and their [Squashtoberfest repo](https://opensource.adobe.com/squashtoberfest/)
 
+#### **Gatsby** (Level 1 swag)
+
+- **Swag**: Pants
```
Comment (crweiner) [inline README.md]: Is there anything that says that Level 2 swag is just pants?

File: README.md [hunk 4]
```diff
@@ -195,6 +206,14 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: Make a PR on [Appwrite Github Repo](https://github.com/appwrite/appwrite) and we'll send you a link.
 - **Notes**: See the [blog post](https://medium.com/appwrite-io/hacktoberfest-2019-is-almost-here-lets-celebrate-it-together-24b311236dd?source=friends_link&sk=104527c24e85006a997fa068bfd1760d) for more information
 
+#### **Gatsby** (Level 1 swag)
+
+- **Swag**: Stickers, T-shirt, Hat
```
Comment (crweiner) [inline README.md]: Is there anything that says that this is for sure the Level 1 swag?

**Comment by crweiner** [comment]
Hay @caioreis123! Thanks for taking the time to add Gatsby! And thank you @travi for helping to review the code!

We pointed out some particular things that need to be fixed ([especilly this one here which appears in multiple places but I only marked it once because I'd like for you to make the changes yourself :smile_cat: ](https://github.com/crweiner/hacktoberfest-swag-list/pull/113/files#r330716156))

Also, please take a moment to make sure that all of the [CodeClimate](https://codeclimate.com/github/crweiner/hacktoberfest-swag-list/pull/113) flags are taken care of and that the [Netlify](https://deploy-preview-113--hacktoberfest-swag-list.netlify.com/) build looks exactly as you think it should. 

Once you've made those changes, let me know and I'll take another look! Thanks!

File: README.md (hunk 3 above)
Comment (caioreis123) [reply README.md]: This issue was the only place I could find with this kind of info

**Comment by crweiner** [comment]
> I think we're all set now, @crweiner. I'm still here for suggestions. This is my first PR ever :)

Hay @caioreis123! I'm very honored that you chose my little project to be your first PR ever! Honestly, thank you for your time and contributions here. It makes me very, very happy to help new people off on the right foot. I hope that this experience was a great one for you that'll give you some more confidence into how to make a great PR no matter the repo. 

Let's get this merged and live! Thank you again for your contribution! And thank you to @travi for helping us both out too!

you are awesome

**Comment by LorisSecuro** [comment]
@crweiner @caioreis123 I'd like to point out that the Gatsby reward system does not seem to be correlated in any way to Hacktoberfest.
So it probably should not be in the list.
Also the linked page (https://www.gatsbyjs.org/contributing/contributor-swag/#how-to-claim-your-free-swag) is not a blog post as indicated, but it's the standard Gatsby contribution guide.

@crweiner if you agree, the PR should be reverted, or if you prefer I can try to remove it myself.

**Comment by crweiner** [comment]
Humm....@LorisSecuro I think you're right. I'll either revert this, or if you'd like to submit a PR, I'll take another look. Thank you for flagging this. I agree that regular dev swag can have a better home elsewhere like [devswag.io](https://devswag.io)

**Pull request merged**